### DISCUSSION TITLE
Catch all errors

### DISCUSSION THREAD

**Discussion by demogit-code** [sqlpage/SQLPage#144] [Q&A]
When you make a mistake the problem is shown on the webpage, referring to files, tables and structure, ...

If what if you wanted to hide these messages (for security or ..), but instead showing a generic error page (without knowing all possible errors in advance) 

So is there a way of hiding these message on the webpage, (or cf like dev vs prod)

Any ideas?

**Comment by lovasoa** [comment]
Hello ! This is not yet possible, but would indeed be very useful for more sensitive deployments. I'm promoting this discussion to an issue to keep track of the work on that.

**Comment by lovasoa** [comment]
I am closing this discussion in favor of : https://github.com/lovasoa/SQLpage/issues/145

If you have some inputs about how you would like to see the feature designed, or if you want to help implementing it, let's discuss on https://github.com/lovasoa/SQLpage/issues/145
